tracer/src/Datadog.Trace/LibDatadog/ResultTag.cs

```diff
@@ -0,0 +1,12 @@
+// <copyright file="ResultTag.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+namespace Datadog.Trace.LibDatadog;
+
+internal enum ResultTag
+{
+    Ok,
+    Err
```

Comment:
If the implied values here (`Ok:0` and `Err:1`) must be consistent with native code, I recommend specifying them explicitly:
```
    Ok = 0,
    Err = 1
```

Otherwise, we run the risk of, for example, someone adding a value in between these two, and then `Err` will be `2` instead of `1`
```
    Ok,    (0)
    Maybe, (1)
    Err    (2)
```